### DISCUSSION TITLE
Get host id

### DISCUSSION THREAD

**Discussion by mdicss** [Dynatrace/dynatrace-configuration-as-code#384] [Archive] [answered]
Is there a way to get a host id or other ids to use them to configure dynatrace with monaco. E.g. I would like to set filters for maintenance windows for some hosts. There I need the host id, not the name.

**Comment by tatomir146** [comment, marked as answer]
Hi @mdicss, there was already some progress of getting these ID's automatically in `monaco`, but in this specific case I would probably try to work using auto-tags or/and management zones. This would give you more flexibility and it's cross environment compatible configuration.

**Reply by mdicss** [reply]
Thanks, tags or management zones are an option, but I fear that we will end up with a huge number of tags.
